lib/backpex/fields/has_many_through.ex

```diff
@@ -414,15 +414,15 @@ defmodule Backpex.Fields.HasManyThrough do
     {:noreply, socket}
   end
 
-  defp action_fields(fields, assigns, action), do: LiveResource.filtered_fields_by_action(fields, assigns, action)
+  defp action_fields(fields, assigns, action), do: LiveResource.fields_by_action(fields, assigns, action)
 
   defp assign_fallback_child_fields(assigns) do
     case Map.has_key?(assigns.field_options, :child_fields) do
       true ->
         assigns
 
       false ->
-        fields = assigns.field_options.live_resource.validated_fields()
+        fields = assigns.field_options.live_resource.fields(:index, assigns)
```

Comment:
I'm not sure about this one.. the fallback child fields will not work because the assigns are the assigns of the field.